test/controllers/crawlers_controller_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+
+class CrawlersControllerTest < ActionController::TestCase
+  setup do
+    @crawler = crawlers(:one)
+  end
+
+  test "should get index" do
+    get :index
+    assert_response :success
+    assert_not_nil assigns(:crawlers)
+  end
+
+  test "should get new" do
+    get :new
+    assert_response :success
+  end
+
+  test "should create crawler" do
+    assert_difference('Crawler.count') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.